bytesource/bytesource.go

```diff
@@ -0,0 +1,61 @@
+package bytesource
```

Comment:
New files need the boilerplate on top, sorry (copy from another file, change the year)

Reply:
A package comment might be good here, too.